### ISSUE TITLE
tests broken in node 10.7.0

### ISSUE THREAD

**Issue by dheffx** [worr/node-imdb-api#66]
Seems the release of node 10.7.0 about a week or so ago has broken the test suite, and Travis is using it for the node10 ci jobs. 

How to recreate:
```
sudo n 10.7.0
npm run check
```

Working tests:
```
sudo n 10.6.0
npm run check
```

Effected files:
```
test/test-getReq.t
test/test-client.t
```

> 1) tests client creation and use
>        merges options from client args:
>      Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/dheffner/dev/node-imdb-api/test/test-client.ts)
> 
> 
>   2) tests client creation and use
>        merges options from client args when performing a search:
>      Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/dheffner/dev/node-imdb-api/test/test-client.ts)
> 
> 
>   3) get
>        times out making a request:
>      Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/dheffner/dev/node-imdb-api/test/test-getReq.ts)
> 
> 
>   4) get
>        times out fetching episodes:
>      Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/Users/dheffner/dev/node-imdb-api/test/test-getReq.ts)
>

**Comment by dheffx** [comment]
i played with it for a few minutes, and got them to pass in 10.7.0 by calling `done();` at the end of each failing test function. however, the tests then were failing on all prior versions of node.

**Comment by worr** [comment]
This is due to https://github.com/nock/nock/issues/1174

**Comment by worr** [comment]
This was fixed in 9baa5b3e3717a1a17a3471aad85cc383f80f63e5, but forgot to follow up here.
